code/index.html

```diff
@@ -1,30 +1,136 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="utf-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Project 3</title>
-    <!-- dont forget to add a css file and link it here! -->
-  </head>
-  <body>
-    <!-- video or image as a header is cool :) -->
-
-    <!-- Signup form -->
-
-    <section class="accordion">
-      <!-- Accordion -->
-      <h1>FAQ</h1>
-
-      <div class="question" id="section1">Section 1</div>
-      <div class="answer">
-        <p>Lorem ipsum...</p>
+
+<head>
+  <meta charset="utf-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Garden family</title>
+  <!-- dont forget to add a css file and link it here! -->
+  <link rel="stylesheet" type="text/css" href="./style.css">
+  <link rel="stylesheet" href="https://use.typekit.net/dyt4sde.css">
+
+
+</head>
+
+<body>
+  <!-- Logo and navbar -->
+  <div>
+    <header>
+      <img class="logo" src="./garden-logo.svg" alt="garden logo">
+
+      <nav>
+        <a href="">News</a>
+        <a href="">Our family</a>
+        <a href="">Our Garden</a>
+        <a href="">About us</a>
+      </nav>
+
+      <div class="hamburger">
+        <div></div>
+        <div></div>
+        <div></div>
       </div>
+    </header>
+
+    <section class="mainSection">
+      <!-- Hero -->
+      <section class="hero-ctn">
+       
+        <div class="heroPic">
+          <img src="./Test-hero.svg" alt="hero pic">
+        <div>
+          <div class="heroText">
+            <h1>Welcome to our garden family</h1>
+          </div>
+      </section>
+  </div>
+
+
+  <!-- Signup form -->
+  <section class="signUp">
+    <div class="container">
+      <div class="signUpContent">
+        <div class="signUpForm">
+          <h2><span class="signUpGreen">Sign up</span> to a wonderful gardening event!</h2>
+          <form class="form" action="https://httpbin.org/anything" method="POST">
+            <label for="name">Name</label>
+            <input type="text" id="name" name="name" required>
+
+            <label for="email">Email</label>
+            <input type="email" id="name" name="email" required>
+
+            <label for="excitedVol" class="excitedVolLabel">How excited are you?</label>
+            <input type="range" id="excitedVol" name="excitedLevel" min="0" max="10" value="5" class="excitedVol">
+
+            <div class="wrapper-row">
+              <label class="checkbox-ctn">
+                <input type="checkbox" checked>
+                <span class="custom-checkbox"></span>
+                Are you ready for a wonderful time?
+              </label>
+            </div>
+
+            <input type="submit" value="Get started" class="getStartedBtn">
 
-      <div class="question" id="section2">Section 2</div>
-      <div class="answer">
-        <p>Lorem ipsum...</p>
+          </form>
+        </div>
+        <div class="signUpImg">
+          <img src="./form-img.svg" alt="">
```

Comment:
Consider using a separate image folder and **always** use the 'alt'-attribute since it's SO important for accessibility!! Here's some reading:
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img